binaries/opae.io/opae/io/utils.py

```diff
@@ -123,9 +123,23 @@ def chown_pci_sva(pci_addr, uid, gid):
         os.chown(sva_bind_dev, uid, gid)
 
 
-def vfio_init(pci_addr, new_owner='', force=False):
+def enable_sriov(enable):
+    sriov = '/sys/module/vfio_pci/parameters/enable_sriov'
+    if os.path.exists(sriov):
```

Comment:
The Linux kernel folks would suggest trying to minimize indentation by something like the following:

if not os.path.exists(sriov):
    LOG.error("Cannot enable sriov because {sriov} does not exist")
    return FALSE

try:...